### ISSUE TITLE
activity styling bug

### ISSUE THREAD

**Issue by grivo** [plantuml/plantuml#1653]
# prerequisites

**plantuml server** version 1.2023.13

# example

```
@startuml
<style>
activityDiagram {
    .x {
        BackGroundColor pink
    }
}
</style>

start

:single line action;
note left : ok

:multi
line
action;
note left : ok

:object; <<task>>
note left
    =question

    is //<<task>>// stereotype the only
    out of the box way to create object
    at activity diagram, btw?
end note

#pink:coloured single line action;
note left : ok

#pink:coloured
multi
line
action; <<x>>
note left : ok

:stereotyped coloured single line action; <<x>>
note left : ok

:stereotyped
coloured
multi
line
action; <<x>>
note left
    =NOK

    once action has multiple
    lines - no styling occurs
end note

stop

@enduml
```

**Comment by The-Lum** [comment]
Hello @grivo,

_[Just for the record]_
See similar reported issue here:
- https://github.com/plantuml/plantuml/issues/1534

Regards,
Th.

**Comment by The-Lum** [comment]
Hello @grivo, and all,

For:
>    Is `<<task>>` stereotype the only
    out of the box way to create object
    at activity diagram, btw?

See this part on the doc.:
- https://plantuml.com/en/activity-diagram-beta#7420c7766a064a28

Then:
- Yes, we can close this issue _(duplicate of #1534)_

_Thanks for the report._
Regards,
Th.

**Comment by The-Lum** [comment]
Duplicate of #1534

**Comment by The-Lum** [comment]
Hi @grivo,

This is now fixed _(by 6c945e25)_
- Thanks @grivo for the report.
- Thanks PlantUML team and @arnaudroques for your work.

Regards.
